client/src/API/API.ts

```diff
@@ -28,9 +30,265 @@ const logout = async () => {
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.856, longitude: 20.225 },
+    },
+    {
+      id: 6,
+      title: "Document 6",
+      description: null,
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.857, longitude: 20.228 },
+    },
+    {
+      id: 7,
+      title: "Document 7",
+      description: "Description 7",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.858, longitude: 20.229 },
+    },
+    {
+      id: 8,
+      title: "Document 8",
+      description: "Description 8",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.859, longitude: 20.23 },
+    },
+    {
+      id: 9,
+      title: "Document 9",
+      description: "Description 9",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.86, longitude: 20.231 },
+    },
+  ];
+}
+
+async function postDocument(id: number, title: string, description: string) {
+  try {
+    const response = await fetch(baseURL + `/documents/${id}`, {
+      method: "POST",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ title, description }),
+    });
+    if (response.ok) {
+      return true;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function patchDocument(id: number, title: string, coordinates: string) {
+  // other fields?
+  try {
+    const response = await fetch(baseURL + `/documents/${id}`, {
+      method: "PATCH",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ title, coordinates }),
+    });
+    if (response.ok) {
+      return true;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function deleteDocument(id: number) {
+  try {
+    const response = await fetch(baseURL + `/documents/${id}`, {
+      method: "DELETE",
+      credentials: "include",
+    });
+    if (response.ok) {
+      return true;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+/****************************   LINKS   **********************************/
+
+async function getLinks(id: number) {
```

Comment:
This route doesn't exist. Check API docs

Reply:
modified